tests/ports/qemu/mpy_arch_flags_test_module/module.py

```diff
@@ -0,0 +1,2 @@
+def hello():
+    print("hello")
```

Comment:
This whole file could just be `hello = 1` if you want to simplify things (up to you).

Also see above comment about possibly removing this file altogether.